MaterialLib/SolidModels/ViscoElasticBurgers-impl.h

```diff
@@ -0,0 +1,332 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef PHYSICALQUANITIES_SOLIDS_VISCOELASTICBURGERS_IMPL_H_
+#define PHYSICALQUANITIES_SOLIDS_VISCOELASTICBURGERS_IMPL_H_
+
+#include <logog/include/logog.hpp>
+
+#include "MaterialLib/SolidModels/KelvinVector.h"
+
+namespace Solids
+{
+template <int DisplacementDim>
```

Comment:
There are references to an article and a book on this model. The implementation is not necessarily a good teaching material.